### DISCUSSION TITLE
Loving this, really nice and simple to set up except snapcast...

### DISCUSSION THREAD

**Discussion by markjsmith** [rern/rAudio#201] [General]
Hi,

I found this nice and easy to set up on my PI, and  lots of features i wanted (Pi4 compatible, etc)
I've tried so many solutions and am getting no where with snapcast, which is frustrating. I just cannot seem to get it to pick up my server, which i can from other snapcast enabled devices (i cannot see rAudio-1 from the snapcast.net app).

When i turn on snapcast client i get this:
```
# pacman -Q snapcast; systemctl status snapclient

snapcast 0.23.0-3
● snapclient.service - Snapcast client
     Loaded: loaded (/usr/lib/systemd/system/snapclient.service; enabled; vendor preset: disabled)
     Active: active (running) since Wed 2021-02-17 14:07:58 UTC; 18min ago
       Docs: man:snapclient(1)
   Main PID: 1604 (snapclient)
      Tasks: 1 (limit: 4915)
     CGroup: /system.slice/snapclient.service
             └─1604 /usr/bin/snapclient --logsink=system --latency=100
```
when i start snapcast, i get the message
``` 
snapcast - sync streaming client
start....
```
but nothing happens! If i go back into `features `at this point, snapcast client is turned off again

I am guessing i need to specify the host address, but i can't work out how/ where. SSH'ing into it didn't seem to help.
Can someone tell me what i should be doing?
Please let me know if you need more details. i have set nothing else up except Spotify connect, but i am guessing this isn't the issue.

**Comment by rern** [comment]
Fixed.
